tests/unit/views/Dashboard/Customer/Home/RequestTest/index.spec.js

```diff
@@ -0,0 +1,45 @@
+import { createLocalVue, shallowMount } from "@vue/test-utils"
+import RequestTest from "@/views/Dashboard/Customer/Home/RequestTest"
+import SelectLocation from "@/views/Dashboard/Customer/Home/RequestTest/SelectLocation"
+import Vue from "vue"
+import Vuetify from "vuetify"
+import Stepper from "@/common/components/Stepper"
+
+Vue.use(Vuetify)
+
+describe("Request Test Page", () => {
+  let container
+  let vuetify
+  let localVue = createLocalVue()
+  
+  beforeEach(() => {
+    vuetify = new Vuetify()
+  })
+
+  it("should render", () => {
+    container = shallowMount(RequestTest, {
+      localVue,
+      vuetify,
+      stubs: {
+        UiDebioStepper: Stepper,
+        SelectLocation,
+      }
+    })
+    
+    expect(container.exists()).toBe(true)
+  })
+
+  it("should render a page with Select Location component", () => {
+    container = shallowMount(RequestTest, {
+      localVue,
+      vuetify,
+      stubs: {
+        UiDebioStepper: Stepper,
```

Comment:
Because there is already a separate test for the stepper, it is probably a good idea to mock the `UiDebioStepper` with a simple `<div></div>` instead.

Reply:
Anyway, you are not testing anything related to the `Stepper` right?